src/api/storage.py

```diff
@@ -50,6 +51,49 @@ def add_datapoints(self, features, arousal_labels, valence_labels):
         self._labels_valence = pd.concat(
             [self._labels_valence, valence_labels])
 
+    def add_songs(self, features, arousal_labels, valence_labels):
+        """
+        Add songs to dataset.
+
+        Args:
+            features (pandas.DataFrame): 2D DataFrame with dimensions
+                [[song_id, sample], features].
+            arousal_labels (np.ndarray): 2D numpy array of arousal values for
+                            the songs in features. Should be same order
+                            with regards to samples as in
+                            the 'features' parameter. Dimensions
```

Comment:
Use \`features\` instead of 'features' to make it look like this: `features`. Same goes for both argument descriptions.